src/Symfony/Component/Asset/Packages.php

```diff
@@ -50,7 +50,7 @@ public function addPackage(string $name, PackageInterface $package)
     /**
      * Returns an asset package.
      *
-     * @param string $name The name of the package or null for the default package
+     * @param string|null $name The name of the package or null for the default package
```

Comment:
please send a PR to update these changes on the right lower branch (same for all your PRs, all grouped in one PR if possible :pray: )